### PR TITLE
Implemented and tested shared modtype ignoring functionality. Updated…

### PR DESCRIPTION
**Pull request by cschroed-usgs**
… disparate uses to leverage new stuff.

### CODE REVIEW

File: nar_common/static/nar_ui/util.js [hunk 1]
```diff
@@ -68,4 +68,11 @@ nar.util = {};
         var msg = errorMsg.replace(/Uncaught .*Error: /, '');
         nar.util.error(msg);
     };
+	nar.util.ignoredModtypes = ['COMP', 'CONTIN'];
+	nar.util.stringContainsIgnoredModtype = function(myString){
```
Comment (cschroed-usgs) [inline nar_common/static/nar_ui/util.js]: http://sugarjs.com/api/Array/some

File: nar_common/static/nar_ui/util.js [hunk 2]
```diff
@@ -68,4 +68,11 @@ nar.util = {};
         var msg = errorMsg.replace(/Uncaught .*Error: /, '');
         nar.util.error(msg);
     };
+	nar.util.ignoredModtypes = ['COMP', 'CONTIN'];
```
Comment (mbucknell) [inline nar_common/static/nar_ui/util.js]: Should ignoredModtypes be capitalized as it is a constant?

**Comment by mbucknell** [comment]
:+1: .Like this refactor!

File: nar_common/static/nar_ui/util.js (hunk 2 above)
Comment (cschroed-usgs) [inline nar_common/static/nar_ui/util.js]: Yes. Good catch. Thanks.

**Pull request merged**